examples/ex9p.cpp

```diff
@@ -551,6 +667,45 @@ FE_Evolution::FE_Evolution(ParBilinearForm &_M, ParBilinearForm &_K,
    M_solver.SetPrintLevel(0);
 }
 
+// Implementation of class FE_Evolution
+FE_Evolution::FE_Evolution(ParBilinearForm &_M, ParBilinearForm &_K,
+                           const Vector &_b, const AIR_parameters &_AIR)
+   : TimeDependentOperator(_M.Height()), b(_b),
+     M_solver(_M.ParFESpace()->GetComm()),
+     z(_M.Height())
+{
+   bool pa = _M.GetAssemblyLevel()==AssemblyLevel::PARTIAL;
+
+   if (pa)
+   {
+      MFEM_ABORT("AIR solver not available for partial assembly.\n");
```

Comment:
The master version uses Jacobi preconditioning in the PA case, and there are sample runs such as `mpirun -np 4 ex9p -pa` which should still work. Can that be restored so that all of the sample runs still work, rather than aborting?

Reply:
This particular if statement is correct. It is in the constructor for using AIR as a preconditioner, which is not compatible with partial assembly. I will try and run the tests though. The new partial assembly that was merged shortly before review began for this changed ex9p and adding AIR became more complicated than it had been

Reply:
I think this can be addressed in conjunction with my other comments about this constructor.